Application/mobile/app_code/lib/l10n/app_en.arb

```diff
@@ -35,6 +36,8 @@
   "contactUs": "Contact us",
   "containersList": "List of containers",
   "containersListEmpty": "No containers available",
+  "createdAt" : "Created at : {time}",
```

Comment:
Remove space before the ':'

Reply:
No, in english the space is needed

Reply:
in french yes there is a space before the ':', but not in english, the same goes for '?' for example